### DISCUSSION TITLE
Support subscan for Humanode project.

### DISCUSSION THREAD

**Discussion by xryoshi** [itering/subscan-essentials#218] [Ideas]
Please support subscan for Humanode,

**Humanode** (https://humanode.io/) is a powerful new-age financial system that leverages the capabilities of the Substrate framework, and a network of cryptographically secure bio-authorized nodes to produce a fair, accessible, and egalitarian platform for decentralized applications and collaborative organizations.

**Comment by yakio** [comment]
Hi @xryoshi , 
Thank you for your suggestion, we are communicating with the Humanode team and following up the cooperation progress!
